dex/testing/dcrdex/harness.sh

```diff
@@ -130,6 +133,20 @@ EOF
 else echo "WARNING: Dogecoin is not running. Configuring dcrdex markets without DOGE."
 fi
 
+if [ $ZEC_ON -eq 0 ]; then
+    cat << EOF >> "./markets.json"
+        },
+        {
+            "base": "ZEC_simnet",
+            "quote": "BTC_simnet",
+            "lotSize": 100000000,
+            "rateStep": 100000,
```

Comment:
This rate step is too high to reflect realistic rates.  No a big deal on simnet, just not realistic.
Currently it's 0.00293146 BTC/ZEC, but I can only use 0.002 or 0.001